src/js/pagination/paging-collection.js

```diff
@@ -48,6 +48,8 @@
                 pageSize: 'page_size',
                 sortKey: 'order_by',
                 order: 'sort_order',
+                totalPages: null,
```

Comment:
I'm confused by this. I thought we don't want these two as query parameters? Also, what's the difference between totalRecords and total_entries?